mediorum/Dockerfile.dev

```diff
@@ -1,12 +1,69 @@
 FROM golang:alpine3.18
 
-RUN apk add build-base make ffmpeg cmake fftw-dev libsndfile-dev git 
+RUN apk add \
+    build-base \
+    make \
+    cmake \
+    fftw-dev \
+    libsndfile-dev \
+    git \
+    curl \
+    eigen-dev \
+    yaml-dev \
+    libsamplerate-dev \
+    taglib-dev \
+    pkgconfig \
+    zlib-dev \
+    chromaprint-dev \
+    nasm \
+    yasm \
+    libogg-dev \
+    libvorbis-dev \
+    libvpx-dev \
+    x264-dev \
+    x265-dev \
+    bash \
+    openssl-dev \
+    bzip2-dev \
+    musl-dev \
+    libffi-dev \
+    lame-dev \
+    gcc
 
-# Required for service registration
-RUN apk add curl build-base python3-dev=3.11.8-r0 py3-pip
-RUN python3 -m pip install --upgrade pip
-RUN python3 -m pip install 'web3==6.6.1'
-RUN python3 -m pip install numpy aubio
+# Install ffmpeg and python from source because essentia is not compatible with the default versions on alpine3.18
+
+# Download and build FFmpeg 4.4
+ARG FFMPEG_VERSION=4.4
+RUN cd /tmp && \
+    wget https://ffmpeg.org/releases/ffmpeg-$FFMPEG_VERSION.tar.bz2 && \
+    tar -xvjf ffmpeg-$FFMPEG_VERSION.tar.bz2 && \
+    cd ffmpeg-$FFMPEG_VERSION && \
+    ./configure --prefix=/usr/local --enable-shared --enable-libx264 --enable-gpl --enable-libmp3lame --enable-avresample && \
+    make -j$(nproc) && \
+    make install && \
+    cd .. && \
+    rm -rf ffmpeg-$FFMPEG_VERSION
+
+# Install Python 3.9 from pyenv
+RUN apk add python3 py3-pip py3-virtualenv bash && \
+    apk add --virtual .build-deps gcc musl-dev libffi-dev openssl-dev bzip2-dev zlib-dev && \
+    cd /tmp && \
+    curl -OL https://www.python.org/ftp/python/3.9.12/Python-3.9.12.tgz && \
+    tar -xzf Python-3.9.12.tgz && \
+    cd Python-3.9.12 && \
+    ./configure --enable-optimizations && \
+    make altinstall && \
+    cd .. && \
+    rm -rf Python-3.9.12 && \
+    apk del .build-deps
+
+# Create and activate a virtual environment
```

Comment:
used a venv bc could not install python3.9 directly on alpine3.18. would've had to downgrade alpine for that. which i tried. and which broke a bunch of other stuff